app/views/admin/products/_account_fields.html.haml

```diff
@@ -1,5 +1,13 @@
 - if SettingsHelper.feature_on? :recharge_accounts
-  = f.input :facility_account_id, collection: current_facility.facility_accounts.active.collect {|fa| [ fa, fa.id ] }, hint: text("hints.deposit_account"), include_blank: false, label: FacilityAccount.model_name.human
+  -# include_blank: false is ignored if you use required: true
+  = f.input :facility_account_id,
+    label: FacilityAccount.model_name.human,
+    label_text: -> (label, _required, _) { "* #{label}" },
+    include_blank: false,
```

Comment:
Ohhhhh my eyes scanning github smushed the `required: true` from line 13 into this form. That makes way more sense.

For the comment, I would maybe explicitly note that you're using `label_text` instead of `required: true` in order to avoid ignoring `include_blank: false`. That's a little more obvious to me.